.github/workflows/e2e-compatibility.yaml

```diff
@@ -0,0 +1,88 @@
+name: Compatibility E2E
+on:
+  workflow_dispatch:
+    release-branch:
+      description: 'Release branch to test'
+      required: true
+      type: choice
+      options:
+        - release/v5.0.x
```

Comment:
this workflow won't become available until it is merged to main, the sample workflow I linked hard coded values rather than receiving them from UI

Reply:
So here we would add other release branches to have them as options in the GitHub UI, right?

Reply:
yes, anything in this list will show up in a drop down menu. There's nothing stopping us to add a new field which allows anything, but I think the dropdown is a nice UX. Perhaps we could add an optional override which would take precedence in the case of wanting to run a custom image.